jck/jck.mk

```diff
@@ -100,14 +100,30 @@ ifndef APPLICATION_OPTIONS
    APPLICATION_OPTIONS :=
 endif
 
+ifneq ($(filter openj9 ibm, $(JDK_IMPL)),)
+	APPLICATION_OPTIONS := customJtx=$(Q)
+	ifneq (,$(wildcard $(JCK_ROOT)$(D)excludes$(D)dev$(D)common.jtx))
+		APPLICATION_OPTIONS += $(JCK_ROOT)$(D)excludes$(D)dev$(D)common.jtx
```

Comment:
- Since Windows supports forward slash `/`, I do not see the benefit of using `$(D)`. 
- `$(JCK_ROOT)/excludes/dev/` should be a parameter. It is used 3 times.

Reply:
Updated.